scripts/check-links.sh

```diff
@@ -127,7 +127,11 @@ check_links() {
 
   # A list of URL patterns that we know are correct but fail to be scraped
   # These are usually sites where anchor links are rendered by Javascript in-browser
-  excludeList=( "github\\.com.*#" "maven\\.apache\\.org.*#" "docs\\.npmjs\\.com.*#" "github\\.com\\/paketo-buildpacks\\/paketo-website\\/edit" )
+  excludeList=( "github\\.com.*#" "maven\\.apache\\.org.*#"
```

Comment:
```suggestion
  excludeList=( "github\\.com.*#"
    "maven\\.apache\\.org.*#"
```

Reply:
Sorry, this diff is a bit confusing. What is that proposed change?